var/spack/repos/builtin/packages/caliper/package.py

```diff
@@ -61,8 +70,9 @@ def cmake_args(self):
         spec = self.spec
 
         args = [
-            '-DBUILD_TESTING=Off',
+            '-DBUILD_TESTING=On',
```

Comment:
This needs to be off by default. Otherwise Caliper will try to download googletest with no way to override. And build a ton of unnecessary stuff.